ASCIIEngine/BasicClasses/GameObject.cs

```diff
@@ -28,23 +31,53 @@ public virtual Material Material
             }
         }
 
-        public virtual int Layer { get; set; }
+        public int Layer { get; set; }
 
-        public virtual bool HasChanged { get; set; }
+        public bool HasChanged { get; set; }
 
         public virtual bool HasCollider { get; set; }
 
         public virtual string Tag { get; set; }
+        
+        public Vector2D Min => Position;
+        
+        public Vector2D Max => Position;
```

Comment:
`Min/Max` should be replaced with `Size`

Reply:
Fixed